pages/common/virsh.md

```diff
@@ -0,0 +1,37 @@
+# virsh
+
+> Manage virsh guest domains.
+> More information: >https://libvirt.org/virshcmdref.html/>.
```

Comment:
If you click on this link it will raise a 404. 
Please update it to the suggestion to fix that. 

```suggestion
> More information: >https://libvirt.org/virshcmdref.html>.
```

Reply:
Fixed the angle bracket in your suggestion 👍

Reply:
Thanks, very confused how that happened...

Reply:
Oh thank @pxgamer.